app/gather/settings.py

```diff
@@ -69,8 +70,16 @@
     if key.startswith(AETHER_PREFIX)
 ]
 
-# ElasticSearch consumer URL
+TENANCY_HEADER = os.environ.get('TENANCY_HEADER', 'X-Oauth-realm')
```

Comment:
To keep the naming rule, could it be `GATEWAY_HEADER_REALM`?
Like:
`GATEWAY_HEADER_TOKEN = os.environ.get('GATEWAY_HEADER_TOKEN', 'X-Oauth-Token')`

Reply:
Another point could be to include this in the django.sdk library...

Reply:
`TENANCY_HEADER` is a consumer sdk variable and the sdk isnt dependent on the django sdk https://github.com/eHealthAfrica/aether-consumer-sdk/blob/master/aet/api.py#L200

Reply:
In that case and to avoid confusion, could you rename it (at least internally) ?
`CONSUMER_TENANCY_HEADER = ...`

Reply:
`TENANCY_HEADER` is already the given for `consumer-sdk, es, ckan` consumers. Naming it differently here will only cause more confusion.

Reply:
Sorry, yesterday I was not explaining it properly, I meant this when I wrote "internally"
`CONSUMERS_TENANCY_HEADER = os.environ.get('TENANCY_HEADER', 'X-Oauth-realm')` (trying to avoid the temptation of using it in Gather for other purposes)